modules/ext.managewiki.oouiform.confirmClose.js

```diff
@@ -28,6 +28,11 @@
 				.not( '#managewiki-submit-reason :input[name]' );
 
 			for ( i = 0; i < $fields.length; i++ ) {
+				if ( $fields[ i ].disabled ) {
+					// Don't count disabled fields
+					continue;
+				}
```

Comment:
I haven't tested if this works, but...
```suggestion
				.not( '#managewiki-submit-reason :input[name]' );
				.not( ':disabled' );

			for ( i = 0; i < $fields.length; i++ ) {
```
(same thing for reviewChanges)